.editorconfig

```diff
@@ -40,6 +40,9 @@ indent_size = 2
 #### C# Formatting Rules ####
 [*.cs]
 
+# CA1848: Use the LoggerMessage delegates
```

Comment:
Editor config was copied from https://github.com/OPCFoundation/UA-.NETStandard/blob/master/.editorconfig i think we should keep it the same

Reply:
Well, that project doesn't use the logger extensions so wouldn't have come across this warning. I want to keep the logger extensions, though, hence I needed to disable the warning.